kernel/solo5.h

```diff
@@ -102,15 +129,17 @@ int solo5_blk_rw(void);
  */
 
 /*
- * Writes (n) bytes from (*buf) to the console. Returns number of bytes
- * actually written.
+ * Writes (size) characters from (*buf) to the console.
+ *
+ * Always succeeds, however the implementation may truncate messages (e.g. due
```

Comment:
Thanks for pointing that out. The backend code could be improved to ignore _some_ errors that are obviously a sign of run-time resource exhaustion (`ENOSPC` comes to mind). For anything else it should probably still assert. In either case I don't see any point in returning a failure status to the guest; what could it reasonably do to "recover"?